source/stylesheets/refills/_progress-bar-indication.scss

```diff
@@ -1,51 +1,50 @@
-.progress-bar-indication {  
-  ///////////////////////////////////////////////////////////////////////////////////
-  $base-border-color: gainsboro !default;
-  $base-border-radius: 3px !default;
+.progress-bar-with-status {
   $base-background-color: white !default;
-  $base-line-height: 1.5em !default;
   $base-accent-color: #477DCA !default;
+  $progress-background-color: darken($base-background-color, 5);
+  $progress-height: 24px;
+  $progress-status-color: $base-accent-color;
+  $progress-width: 100%;
+  $progress-border-radius: $progress-height;
 
-  p {
-    line-height: $base-line-height;
-  }
-  //////////////////////////////////////////////////////////////////////////////////
+  position: relative;
 
-  $progress-border-color: $base-border-color;
-  $progress-border: 1px solid $progress-border-color;
-  $progress-meter-border-color: $base-accent-color;
-  $progress-meter-border: 1px solid darken($progress-meter-border-color, 15%);
-  $progress-meter-color: $progress-meter-border-color;
-  $progress-background: darken($base-background-color, 5);
-  $progress-animation-duration: 0.7s;
-  $progress-color: white;
+  .progress-bar-indication {
+    @include appearance(none);
+    background-color: $progress-background-color;
+    border-radius: $progress-border-radius;
```

Comment:
Properties should be ordered background-color, border, border-radius, color, display, height, width